core/common/src/main/java/alluxio/conf/InstancedConfiguration.java

```diff
@@ -204,6 +204,31 @@ public void set(PropertyKey key, Object value) {
     set(key, value, Source.RUNTIME);
   }
 
+  /**
+   * Sets the value for the appropriate key in the {@link Properties}.
+   *
+   * @param key the key to set
+   * @param value the value for the key
+   */
+  @java.lang.Deprecated
```

Comment:
Why deprecated this method, would you like to add a description into the method java doc comment?